docs-v2/reference/sdks/node.mdx

```diff
@@ -1255,6 +1255,87 @@ const { data } = await nango.createConnectSession({
 </Expandable>
 
 
+### Create a reconnect session
```

Comment:
maybe we should explain what a reconnect session means or when it should be used? wdyt?

Reply:
Yes I can add a comment 👌🏻